src/openqaoa-azure/backends/devices.py

```diff
@@ -28,9 +28,9 @@ def __init__(self, device_name: str, resource_id: str, az_location: str):
         device_name: `str`
             The name of the Azure remote QPU/Simulator to be used
         resource_id: `str`
-            resource_id
+            The resource_id of the Workplace
         az_location: `str`
-            az_location
+            The location of the Azure Workplace. e.g. "us-west-1"
```

Comment:
Are you sure they write the locations in that way? I have only seen it like `eastus`

Reply:
You're right. The location example here is from braket. ill change that